scripts/build.sh

```diff
@@ -86,17 +132,18 @@ ${GOX:?command not found} \
 set -e
 
 # trim GOPATH to first element
-IFS="$PATHSEP"
-MAIN_GOPATH=($GOPATH)
-MAIN_GOPATH="$(convert_path --unix "$MAIN_GOPATH")"
-IFS=$OLDIFS
+IFS="${PATHSEP}"
+# FIXME: How do you know that the first path of GOPATH is the main GOPATH? Or is the main GOPATH meant to be the first path in GOPATH?
```

Comment:
I'm unsure whether this is about the "first" go path or some kind of "main" path. I'm not familiar with go conventions...